src/ui/page/subscriptions/view.jsx

```diff
@@ -1,105 +1,55 @@
+    doFetchRecommendedSubscriptions,
+    suggestedSubscriptions,
+    loading,
+  } = props;
+  const hasSubscriptions = !!subscribedChannels.length;
+  const [showSuggested, setShowSuggested] = useState(!hasSubscriptions);
+
+  useEffect(() => {
     doFetchMySubscriptions();
     doFetchRecommendedSubscriptions();
-
-    // For channels that already have subscriptions, show the suggested subs right away
-    // This can probably be removed at a future date, it is just to make it so the "view your x subscriptions" button shows up right away
-    // Existing users will still go through the "first run"
-    if (!firstRunCompleted && allSubscriptions.length) {
-      doShowSuggestedSubs();
-    }
-  }
-
-  onAutoDownloadChange(event: SyntheticInputEvent<*>) {
-    this.props.doSetClientSetting(SETTINGS.AUTO_DOWNLOAD, event.target.checked);
-  }
-
-  render() {
-    const {
-      subscribedChannels,
-      allSubscriptions,
-      loading,
-      autoDownload,
-      viewMode,
-      doSetViewMode,
-      loadingSuggested,
-      firstRunCompleted,
-      doCompleteFirstRun,
-      doShowSuggestedSubs,
-      showSuggestedSubs,
-      unreadSubscriptions,
-    } = this.props;
-    const numberOfSubscriptions = subscribedChannels && subscribedChannels.length;
-
-    return (
-      // Only pass in the loading prop if there are no subscriptions
-      // If there are any, let the page update in the background
-      // The loading prop removes children and shows a loading spinner
-      <Page notContained loading={loading && !subscribedChannels} className="main--no-padding-top">
-        {firstRunCompleted ? (
-          <UserSubscriptions
-            viewMode={viewMode}
-            doSetViewMode={doSetViewMode}
-            hasSubscriptions={numberOfSubscriptions > 0}
-            subscriptions={allSubscriptions}
-            autoDownload={autoDownload}
-            onChangeAutoDownload={this.onAutoDownloadChange}
-            unreadSubscriptions={unreadSubscriptions}
-            loadingSuggested={loadingSuggested}
-          />
-        ) : (
-          <FirstRun
-            showSuggested={showSuggestedSubs}
-            doShowSuggestedSubs={doShowSuggestedSubs}
-            loadingSuggested={loadingSuggested}
-            numberOfSubscriptions={numberOfSubscriptions}
-            onFinish={doCompleteFirstRun}
-          />
-        )}
-      </Page>
-    );
-  }
+  }, [doFetchMySubscriptions, doFetchRecommendedSubscriptions]);
+
+  return (
+    <Page>
+      <div className="card">
+        <FileList
+          loading={loading}
+          sort={
+            <Button
+              button="alt"
+              label={showSuggested ? hasSubscriptions && __('View Your Subscriptions') : __('Find New Channels')}
+              onClick={() => setShowSuggested(!showSuggested)}
+            />
+          }
+          header={<h1>{showSuggested ? __('You might like these channels') : __('Latest From Your Subscriptions')}</h1>}
```

Comment:
why does the capitalization switch?

Reply:
Because someone stole my computer and changed the titles without me knowing.